modules/statics/src/coins.ts

```diff
@@ -7111,6 +7119,30 @@ export const coins = CoinMap.fromCoins([
     '0xe9e7cea3dedca5984780bafc599bd69add087d56',
     UnderlyingAsset['bsc:busd']
   ),
+  bscToken(
+    '2400c915-d610-45e7-bd7d-a028eb653b42',
+    'bsc:usdt',
+    'Tether USD',
+    18,
+    '0x55d398326f99059ff775485246999027b3197955',
+    UnderlyingAsset['bsc:usdt']
+  ),
+  bscToken(
+    '95296300-6d0b-4011-9028-53f54750d4b6',
+    'bsc:ksm',
+    'Kusama (Binance Pegged)',
+    5,
+    '0xe646c8fc9f507529b05fe0a825ae9beb1aad9f6b',
+    UnderlyingAsset['bsc:ksm']
+  ),
+  bscToken(
+    '82e8167f-2c11-46b9-b7dd-eab667d8429b',
+    'bsc:vet',
+    'VeChain (Binance Pegged)',
```

Comment:
```suggestion
    'VeChain Token',
```

Reply:
@mmcshinsky-bitgo same as KSM; this is the binane pegged one and we want to make it very clear that this is NOT the native L1 VeChain Token 
cc @evalenti-simtlix